Pages/List/list.html

```diff
@@ -66,6 +66,14 @@ <h3>Numbered List - I</h3>
                 <li>list Item3</li>
             </ol>
         </div>
+        <p class="body-r">Copy the given code to create Numbered List - I</p>
+
+        <iframe
+  src="https://carbon.now.sh/embed?bg=rgba%283%2C26%2C77%2C1%29&t=blackboard&wt=none&l=htmlmixed&width=680&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Fira+Code&fs=13px&lh=152%25&si=false&es=2x&wm=false&code=%253Col%253E%250A%2509%253Cli%253Elist%2520Item1%253C%252Fli%253E%250A%2509%253Cli%253Elist%2520Item2%253C%252Fli%253E%250A%2509%253Cli%253Elist%2520Item3%253C%252Fli%253E%250A%253C%252Fol%253E"
+  style="width: 50rem; height: 40rem; border:0; transform: scale(1); overflow:hidden;"
```

Comment:
You can remove the inline styles, and may be give a class and extract this out, for code iframe tags.